src/aiortc/contrib/media.py

```diff
@@ -434,24 +490,84 @@ async def stop(self):
 
     async def __run_track(self, track, context):
         while True:
+            # if self.frame_width is not None and self.frame_height is not None:
+            #     if (track.kind == "keypoints" and enable_prediction == True) or \
+            #     (track.kind == "video" and enable_prediction == False):
```

Comment:
Do these comments need to stay here? If not, can we please remove?

Reply:
Removed them